bedrock/firefox/templates/firefox/new/scene1.html

```diff
@@ -51,31 +45,17 @@
                 {{_('<a href="%(url)s">Update</a> your Firefox for the latest in speed and privacy.')|format(url='https://support.mozilla.org/kb/update-firefox-latest-version') }}
               </p>
               <p id="version-message-desktop-pre-release" class="version-message">
-                {% if l10n_has_tag('firefox_new_pre_release') %}
-                  {{_('You’re using a pre-release version of Firefox.') }}
-                {% else %}
-                  {{_('Congrats! You’re using the latest version of Firefox.')}}
-                {% endif %}
+                {{_('You’re using a pre-release version of Firefox.') }}
               </p>
             </div>{#-- /.version-message-container --#}
           </header>
           <div class="main-content">
-            <h1>{{ _('Browse Freely') }}</h1>
+            <h1>{{ _('Chart your own course') }}</h1>
             <section id="download-buttons">
               {# handles desktop users (windows, os x, linux) #}
               <div class="download-button-wrapper desktop" id="download-button-wrapper-desktop">
-                {# **WARNING**
-
-                  The `locale_in_transition` parameter should be used very carefully. It's included
-                  here because this template and scene2 share a lang file, and are therefore
-                  translated into the same list of locales. Adding this to another download button
-                  would significantly restrict the builds of Firefox available for download.
-
-                  Bug 1290962
-                #}
```

Comment:
Is this warning no longer valid?

Reply:
It most ceetainly is valid, I'll re-add it to the new default template